### PR TITLE
Add email field to API

### PR DESCRIPTION
**Pull request by didrikmunther**
## Describe your changes

Fixes: #805 by adding an `armada_email` field to profiles, and exposing this field in the API call `/api/organization`.

## Checklist before requesting review

- [x] Feature/fix PRs should add one atomic (as small as possible) feature or fix.
- [x] All code in your PR needs to have been formatted.
- [x] The merge commit title should be prefixed with the type of the PR and a colon and a space, that is "feat", "fix", "doc", "chore", or "refactor", followed by a ": ".
- [x] The merge commit body should reference at least one issue.
- [x] The code compiles and all the tests pass.

### CODE REVIEW

File: templates/people/profile.html [hunk 1]
```diff
@@ -21,6 +21,7 @@ <h3 class="panel-title">Basic information</h3>
 		<div class="panel-body">
 			<ul class="list-unstyled">
 				<li><span style="font-weight: 700;">E-mail address:</span> <a href="mailto:{{ profile.user.email }}">{{ profile.user.email }}</a></li>
+				<li><span style="font-weight: 700;">Armada e-mail address:</span> {% if profile.armada_email %}<a href="tel:{{ profile.armada_email }}">{{ profile.armada_email }}</a>{% else %}<span style="font-style: italic;">not given</span>{% endif %}</li>
```
Comment (Hampfh) [inline templates/people/profile.html:24]: why is it `tel:` and not `mailto:`?

**Review by Hampfh — COMMENTED** [review]
Nice work, will it also be possible to edit the email? I see no logic for mutating it but maybe Django does that under the hood?

File: templates/people/profile.html (hunk 1 above)
Comment (didrikmunther) [reply templates/people/profile.html]: That's a mistake, fixed

**Review by Hampfh — APPROVED** [review]

**Comment by didrikmunther** [comment]
> Nice work, will it also be possible to edit the email? I see no logic for mutating it but maybe Django does that under the hood?

Yes, Django automagically adds functionality for mutating the field in both the admin page, and the edit profile page.

**Review by Hampfh — APPROVED** [review]

**Pull request merged**